models/notification/src/index.ts

```diff
@@ -41,6 +41,29 @@ export class TNotification extends TAttachedDoc implements Notification {
   status!: NotificationStatus
 }
 
+@Model(notificaton.class.EmaiNotification, core.class.Doc, DOMAIN_NOTIFICATION)
+export class TEmaiNotification extends TDoc implements EmaiNotification {
+  @Prop(TypeString(), 'Sender' as IntlString)
```

Comment:
Could we make them as notification.string.Sender?

Reply:
Currently I am doing #763 and it will be fixed